stock_no_negative/views/product_product_views.xml

```diff
@@ -11,7 +11,7 @@
         <field name="inherit_id" ref="stock.view_template_property_form" />
         <field name="arch" type="xml">
             <field name="categ_id" position="after">
-                <field name="allow_negative_stock" invisible="type != 'product'" />
+                <field name="allow_negative_stock" invisible="type == 'service'" />
```

Comment:
`not is_storable`, I think. See https://github.com/odoo/odoo/commit/728d9f83f6d14

Reply:
Also, might be a nice improvement to move the field until after `is_storable` itself.
![image](https://github.com/user-attachments/assets/1ab38694-187e-4d7a-888a-562b8190cb6c)